plenum/test/waits.py

```diff
@@ -107,7 +107,7 @@ def expectedClientRequestPropagationTime(nodeCount):
 
 
 def expectedTransactionExecutionTime(nodeCount):
-    return int(CLIENT_REPLY_TIMEOUT * nodeCount)
+    return int(2 * CLIENT_REPLY_TIMEOUT * nodeCount)
```

Comment:
This timeout is already huge, it was a mistake, `CLIENT_REPLY_TIMEOUT` does not mean the timeout factor for a single node, it means the time it should take for the client to get a reply, its considering an average pool size, lets not increase that because it will make some tests too slow since they expect to never receive a reply.